proto/src/determined/api/v1/trial.proto

```diff
@@ -595,6 +595,8 @@ message AcceleratorData {
   string accelerator_type = 5;
   // An array of UUIDs of the accelerators associated with the allocation.
   repeated string accelerator_uuids = 6;
+  // The name of the resource pool.
+  string resource_pool = 7;
```

Comment:
its a little awkward to me that we won't use this field when calling ```v1PostAllocationAcceleratorDataRequest``` but I think this is fine